src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -41,9 +47,11 @@ export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
       >
         {!hasVenues && (
           <>
-            <div className="admin-venue__title">Welcome!</div>
             <div className="admin-venue__title">
-              Create your first Sparkle space
+              <M>{AdminVenues.name}.welcome.message</M>
```

Comment:
It could be useful if there was a `code` prop in addition for idioms like `<M code="some.code'" />`